src/core/sys/posix/sys/shm.d

```diff
@@ -121,6 +121,31 @@ else version( FreeBSD )
     int   shmdt(in void*);
     int   shmget(key_t, size_t, int);
 }
+else version( DragonFlyBSD )
```

Comment:
Neither is this.

Reply:
Ooh... but it should be (will amend dlang/druntime). Thanks for noticing this one !

Reply:
Link: https://github.com/dlang/druntime/pull/2106

Reply:
dlang#2106 has been approved => waiting to be merged